schema.sql

```diff
@@ -10,4 +10,33 @@
 
```

Comment:
- [ ] Please in this code I see that you try to make the changes requested in the requirments but I think that you forget that you deleted the species column from the animal table in this line as requested so it would be better to chagne the way of updating the animals you should set the species_id to the digimon you should make a query to get the the species_id for the an specie called digimon   `UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';`  it should look something like this ```sql
   UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Digimon')
   WHERE name like '%mon';
```.
,Please I think you did the same mistake in this code as will 
UPDATE animals SET species = 'pokemon' WHERE species IS NULL OR species = '';
 you should try to use the last approach on this one as will  .if an animal's name ends with `mon` it should have `Digimon` as a `species_id`, all other animals should have `Pokemon` instead